cmd/ipfs/init.go

```diff
@@ -142,6 +148,23 @@ func addTheWelcomeFile(conf *config.Config) error {
 	return nil
 }
 
+func initializeIpnsKeyspace(conf *config.Config) error {
+	ctx, cancel := context.WithCancel(context.Background())
+	nd, err := core.NewIPFSNode(ctx, core.Offline(conf))
+	if err != nil {
+		return err
+	}
+	defer nd.Close()
```

Comment:
Also, seems like initing ipfs only needs the dagservice and routing, so may not need the whole node-- @briantigerchow knows best as he's been refactoring all that.